src/components/DocumentScanner.tsx

```
@@ -0,0 +1,272 @@
+  const theme = useTheme();
+  const toast = useToast();
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const [selectedDocument, setSelectedDocument] = useState<Document | null>(null);
+  const [documents, setDocuments] = useState<Document[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const { updateUserData } = useContext(AuthContext)!;
+
+  useEffect(() => {
+    const fetchDocuments = async () => {
+      try {
+        const response = await getDocumentsList();
```

ESLint

[error] Replace `········` with `↹↹↹↹`

(prettier/prettier)